source/_integrations/dsmr.markdown

```diff
@@ -1,15 +1,16 @@
 ---
 title: DSMR Slimme Meter
 description: Instructions on how to integrate DSMR Smartmeter within Home Assistant.
-logo: netbeheernederland.jpg
+logo: netbeheernederland.png
```

Comment:
Please add a new brand logo to his repository -> https://github.com/home-assistant/brands

Reply:
Just double checking, I already added brand logo's: https://github.com/home-assistant/brands/pull/1739
So basically if I remove the logo item here it'll automatically pick up from the brands repository?

I also removed the jpg and added a new logo as png in the supported_brands folder, Should I just remove the jpg then from this folder? I guess it's nowhere used anymore after removing the line.

Reply:
Yes, in that case you only have to delete this line and the .png file, everything is retrieved from the brand repository 😉

Reply:
Ok, clear, removed it